pfcpiface/parse-sdf.go

```
@@ -128,7 +120,7 @@ func parseFlowDesc(flowDesc, ueIP string) (*ipFilterRule, error) {
 	}
 
 	ipf.direction = fields[1]
-	ipf.proto = parseProto(fields[2])
+	ipf.proto, _ = parseL4Proto(fields[2])
```

Ignore errors here for now